ansible/inventory/production/group_vars/all/vars.yml

```diff
@@ -1,5 +1,5 @@
 # Minimun required Ansible version
-ansible_version_min: 2.8
+ansible_version_min: 2.10.0
```

Comment:
the testing server is higher (`ansible_version_min: 2.10.1`). Is that for any particular reason?

We should try to have the lowest ansible version possible that still supports all features, I think

Reply:
Please see my latest commit 71e085f.